gst/tensor_merge/gsttensormerge.c

```diff
@@ -602,7 +602,9 @@ gst_tensor_merge_collected (GstCollectPads * pads,
   gst_buffer_append_memory (tensor_buf, mem);
   gst_buffer_copy_into (tensor_buf, tensors_buf, GST_BUFFER_COPY_TIMESTAMPS, 0,
       -1);
-  gst_buffer_unref (tensors_buf);
+
+  if (tensors_buf)
```

Comment:
please do rollback this change, it may cause another coverity issue.

Reply:
Done.